requirements-dev.txt

```diff
@@ -273,7 +273,7 @@ snowflake-connector-python==3.9.0 ; python_version >= "3.9" and python_version <
 sortedcontainers==2.4.0 ; python_version >= "3.9" and python_version < "4.0"
 soupsieve==2.5 ; python_version >= "3.10" and python_version < "3.13"
 sphobjinv==2.3.1 ; python_version >= "3.10" and python_version < "3.13"
-sqlalchemy==2.0.29 ; python_version >= "3.9" and python_version < "4.0"
+sqlalchemy==2.0.29 ; python_version >= "3.10" and python_version < "3.13"
```

Comment:
We still depend on sqlalchemy **for development dependencies only** by way of `jupyter-cache`.